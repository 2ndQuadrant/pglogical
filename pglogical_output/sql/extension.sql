CREATE EXTENSION pglogical_output;

SELECT pglogical_output_proto_version();

SELECT pglogical_output_min_proto_version();

DROP EXTENSION pglogical_output;
