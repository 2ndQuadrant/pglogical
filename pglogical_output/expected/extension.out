CREATE EXTENSION pglogical_output;
SELECT pglogical_output_proto_version();
 pglogical_output_proto_version 
--------------------------------
                              1
(1 row)

SELECT pglogical_output_min_proto_version();
 pglogical_output_min_proto_version 
------------------------------------
                                  1
(1 row)

DROP EXTENSION pglogical_output;
